package.json

```diff
@@ -5,6 +5,7 @@
   "main": "src/Repl.svelte",
   "svelte": "src/Repl.svelte",
   "scripts": {
+    "start": "rollup -w -c demo/rollup.config.js",
```

Comment:
Going to suggest that we prepend `npm run srcdoc` to this too, since it doesn't work without it.